app/controllers/admin/invoices_controller.rb

```diff
@@ -6,5 +6,6 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
     @invoice_items = @invoice.invoice_items.includes(:item)
+    @total_revenue = @invoice.invoice_items.sum { |item| item.quantity * item.unit_price}
```

Comment:
I think this should be a method in the Invoice model using Active Record!

Reply:
Good idea!